project_one_Brian-Jones/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+    <title>Placeholder</title>
+
+</head>
+
+<body>
+    <h1>Header for big things that are to follow</h1>
+
+    <div>
+        <p>Press me to make back things happen: <button id="grading-element-one" onclick="baddie()">DO IT</button></p>
+        
+    </div>
+
+    <div id="container">
+
+        <a href="/about.html" target="_blank"><button id="grading-element-two">About page </button></a>
+
+        <p>Click me to fill something out <button id="grading-element-three" onclick="unHide()">Press me</button></p>
+
+
+        <div id="grading-element-three-consequence" hidden>
```

Comment:
almost works as expected. right now you are changing `display: none` to `display: block`...can you make it start as `visibility: hidden` to `visibility: visible`